Client/Telemetry/TelemetryWrapper.swift

```diff
@@ -359,6 +359,7 @@ extension TelemetryWrapper {
         case cancel = "cancel"
         case change = "change"
         case close = "close"
+        case displayed = "displayed"
```

Comment:
Instead of adding a new event here we can use `view` instead. These events don't add any real value as only the what is send through Glean counts.